app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -136,16 +136,19 @@ function trackCustomImpressions() {
     }
 
     // display add
-    var adBox = document.getElementById('sponsorship-arbitrary-display-widget')
-    if (adBox
+    var rotatingDisplayUnits = document.querySelectorAll('[data-rotating-display-unit]');
```

Comment:
I assume this means "rotating" as in "regularly changing"? Because at first I was slightly confused by this and thought about animation. Maybe there's a better name for this that would avoid the same confusion for others?

Reply:
+1 on this, I also thought about an animation when I read this 😅

Reply:
Changed to `data-display-unit`... which really does make a lot more sense.